# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../qemu-kvm/qemu-kvm-0.14.1-progname.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- qemu-kvm-0.14.1/Makefile.target.orig	2011-06-01 19:42:21.004004948 +0200
+++ qemu-kvm-0.14.1/Makefile.target	2011-06-01 19:43:01.144004948 +0200
@@ -24,9 +24,9 @@
 else
 # system emulator name
 ifeq ($(TARGET_ARCH), i386)
-QEMU_PROG=qemu$(EXESUF)
+QEMU_PROG=qemu-kvm$(EXESUF)
 else
-QEMU_PROG=qemu-system-$(TARGET_ARCH2)$(EXESUF)
+QEMU_PROG=qemu-kvm-system-$(TARGET_ARCH2)$(EXESUF)
 endif
 endif
 
